cli/src/main/java/com/devonfw/tools/ide/tool/lazydocker/LazyDocker.java

```diff
@@ -0,0 +1,70 @@
+package com.devonfw.tools.ide.tool.lazydocker;
+
+import com.devonfw.tools.ide.common.Tag;
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.process.ProcessContext;
+import com.devonfw.tools.ide.process.ProcessErrorHandling;
+import com.devonfw.tools.ide.process.ProcessMode;
+import com.devonfw.tools.ide.process.ProcessResult;
+import com.devonfw.tools.ide.tool.LocalToolCommandlet;
+import com.devonfw.tools.ide.tool.ToolCommandlet;
+import com.devonfw.tools.ide.version.VersionIdentifier;
+
+import java.util.Set;
+
+/**
+ * {@link ToolCommandlet} for <a href="https://github.com/jesseduffield/lazydocker">lazydocker</a>.
+ */
+public class LazyDocker extends LocalToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public LazyDocker(IdeContext context) {
+
+    super(context, "lazydocker", Set.of(Tag.DOCKER));
+  }
+
+  @Override
+  public boolean install(boolean silent) {
+
+    String bashPath = this.context.findBash();
+    String command = "docker version --format '{{.Client.APIVersion}}'";
+    ProcessContext pc = this.context.newProcess().errorHandling(ProcessErrorHandling.NONE).executable(bashPath)
+        .addArgs("-c", command);
+    ProcessResult result = pc.run(ProcessMode.DEFAULT_CAPTURE);
+
+    if (result.getOut().isEmpty()) {
+      this.context.info("Docker is not installed, but required for lazydocker.");
+      this.context.info("To install docker, call the following command:");
+      this.context.info("ide install docker");
+      return false;
+    }
+
+    VersionIdentifier dockerAPIversion = VersionIdentifier.of(result.getOut().get(0).toString());
+
+    if (dockerAPIversion.compareVersion(VersionIdentifier.of("1.25")).isLess()) {
+      this.context.info("The installed version of docker does not meet the requirements of lazydocker.");
+      this.context.info("Please upgrade your installation of docker, before installing lazydocker.");
+      return false;
+    }
+
+    command = "docker-compose version --short";
+    pc = this.context.newProcess().errorHandling(ProcessErrorHandling.NONE).executable(bashPath).addArgs("-c", command);
+    result = pc.run(ProcessMode.DEFAULT_CAPTURE);
+
+    if (!result.getOut().isEmpty()) {
+      VersionIdentifier dockercomposeversion = VersionIdentifier.of(result.getOut().get(0).toString());
+
+      if (dockercomposeversion.compareVersion(VersionIdentifier.of("1.23.2")).isLess()) {
+        this.context.info("Found docker-compose version:" + dockercomposeversion);
+        this.context.info(
+            "If you want to use docker-compose with lazydocker, then you'll need at least version 1.23.2 of docker-compose");
+      }
+    }
+
+    return super.doInstall(silent);
+  }
+}
```

Comment:
If I remember correctly, we had some fun to properly run lazydocker on Windows and had to invent some workarounds.
IMHO this is missing here:
https://github.com/devonfw/ide/blob/cdf4af5de21489f3fdcbfdab41af858822e0d598/scripts/src/main/resources/scripts/command/lazydocker#L22-L27

Reply:
Is the intention to start lazydocker in a seperate powershell window? I had tested running lazydocker on Windows, and it worked just fine for me.

Reply:
Then maybe something changed meanwhile. There was some problem on Windows that it locked or ended the bash terminal or so. I do not really remember. We can tomorrow see if someone in the team remembers. Otherwise, we can also merge as is if you have already tested this on Windows it should be fine and we do not need such powershell workaround...

Reply:
Since we could not really dig out something more here in our daily, we can merge as is.
If there is a bug, we can fix later...